dcos_installer/upgrade.py

```diff
@@ -74,13 +74,17 @@
 # Check if the node has node/cluster checks and run them
 if [ -f /opt/mesosphere/etc/dcos-diagnostics-runner-config.json ]; then
    # command exists
-   if !(output=$(dcos-diagnostics check node-poststart)); then
+   output=$(dcos-diagnostics check node-poststart)
+   ret=$?
+   if [ $ret -ne 0 ]; then
```

Comment:
Ditto - can we remove the assignment?

Looking below in this file, we follow the same pattern of assignment as used here. Local consistency is nice - maybe either go with the existing style (the other occurrence uses `$RETCODE`), or add another commit to this PR which updates the other site as well?